internal/tui/note.go

```diff
@@ -15,25 +15,66 @@ var InfoHeader = Str{
 	Fancy: aec.Bold.Apply(aec.LightCyanB.Apply(aec.BlackF.Apply("i")) + " " + aec.LightCyanF.Apply("Info → ")),
 }
 
-func (o Output) PrintNote(format string, args ...any) {
+type options struct {
+	header       Str
+	messageColor aec.ANSI
+}
+
+type NoteOptions func(o *options)
+
+func WithMessageColor(color aec.ANSI) NoteOptions {
+	return func(o *options) {
+		o.messageColor = color
+	}
+}
+
+func WithHeader(header Str) NoteOptions {
+	return func(o *options) {
+		o.header = header
+	}
+}
+
+func (o Output) PrintNoteWithHeader(format string, args []any, opts ...NoteOptions) {
 	if o.isTerminal {
 		// TODO: Handle all flags
 		format = strings.ReplaceAll(format, "--platform", ColorFlag.Apply("--platform"))
 	}
 
-	header := o.Sprint(InfoHeader)
+	opt := &options{
+		header:       InfoHeader,
+		messageColor: noColor{},
+	}
 
-	_, _ = fmt.Fprint(o, "\n", header)
+	for _, override := range opts {
+		override(opt)
+	}
+
+	h := o.Sprint(opt.header)
+
+	_, _ = fmt.Fprint(o, "\n", h)
 	s := fmt.Sprintf(format, args...)
 	for idx, line := range strings.Split(s, "\n") {
 		if idx > 0 {
-			_, _ = fmt.Fprint(o, strings.Repeat(" ", Width(header)))
+			_, _ = fmt.Fprint(o, strings.Repeat(" ", Width(h)))
 		}
 
 		l := line
 		if o.isTerminal {
-			l = aec.Italic.Apply(l)
+			l = opt.messageColor.Apply(aec.Italic.Apply(l))
 		}
 		_, _ = fmt.Fprintln(o, l)
 	}
 }
+
+func (o Output) PrintNote(format string, args ...any) {
+	o.PrintNoteWithHeader(format, args, WithHeader(InfoHeader))
+}
+
+var WarningHeader = Str{
```

Comment:
Looks like this one is only used internally by `PrintWarning` for now, so we could consider keeping it non-exported